### PR TITLE
fix(sidenav): add min width for empty navs

### PR DESCRIPTION
**Pull request by kara**
r: @hansl @jelbourn

### CODE REVIEW

**Comment by jelbourn** [comment]
LGTM

File: src/components/sidenav/sidenav.scss [hunk 1]
```diff
@@ -107,6 +107,7 @@ $md-sidenav-push-background-color: md-color($md-background, dialog) !default;
     top: 0;
     bottom: 0;
     z-index: 3;
+    min-width: 5%;
```
Comment (DevVersion) [inline src/components/sidenav/sidenav.scss]: Hm, just questioning -  I don't think that using a percentage is good here. `5%` sounds like a random percentage. 
- Percentage in relative to the parent, do you think that looks / works good on small screens?

Comment (hansl) [inline src/components/sidenav/sidenav.scss]: That's just in case there's no content in the sidenav. Likely

**Comment by hansl** [comment]
LGTM.

**Comment by sunderw** [comment]
Hi, this CSS rule is causing an overflow on the X axis on my page. I don't really understand which issue it is supposed to fix. Could someone enlighten me ?

**Pull request merged**